docker-compose.yml

```diff
@@ -55,3 +58,8 @@ services:
     ports:
       - 4567
 
+  stripe:
+    image: stripemock/stripe-mock:latest
+    container_name: stripe
+    ports:
+      - "$STRIPE_API_PORT:12112"
```

Comment:
A note, `12112` is the default, exposed port by Stripe Mock.  To change this requires extra command line parameters and thusly this port was not extracted to configuraiton.